src/mobile/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "mobile",
+  "version": "1.0.0",
+  "main": "node_modules/expo/AppEntry.js",
```

Comment:
Can you explain this?

Reply:
Well, it's definetly not version 1.0 of our app 😄 Expo-cli creates this by default. I can change it to 0.1 or it can be done next time someone works on the app.

Reply:
Oh wrong line. package.json was auto generated whit `expo init` but you are right there's no need to include it. Just tested again and it works without it.